### PR TITLE
Do not hardcode the libexecdir to /usr/libexec

### PR DESCRIPTION
**Pull request by hughsie**
Fixes https://github.com/fwupd/fwupd/issues/4360

Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [X] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: data/installed-tests/fwupdmgr-p2p.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@ if [ "$(id -u)" -ne 0 ]; then exit 0; fi
 echo "Starting P2P daemon..."
 export FWUPD_DBUS_SOCKET="/var/run/fwupd.sock"
```
Comment (superm1) [inline data/installed-tests/fwupdmgr-p2p.sh:11]: If you're processing the file anyway shouldn't this be swapped out by configuration variable too?

Comment (hughsie) [reply data/installed-tests/fwupdmgr-p2p.sh:11]: ohh hmm. Do we have any config for the socket location now? In the common code we have:

	/* /run/lock */
	case FU_PATH_KIND_LOCKDIR:
		return g_strdup("/run/lock");

Comment (superm1) [reply data/installed-tests/fwupdmgr-p2p.sh:11]: I think specifically systems directory to use for configuration (/run) can it be discovered using systemd pkgconfig?

Comment (hughsie) [reply data/installed-tests/fwupdmgr-p2p.sh]: The twitterati seems to think that we should just hardcode /run now. I've made that change as a followup.

**Review by superm1 — APPROVED** [review]

**Pull request merged**